<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cqp::DeviceEditor</class>
 <widget class="QDialog" name="cqp::DeviceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Device</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QLineEdit" name="switchName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="siteAgentLbl">
     <property name="text">
      <string>Site Agent</string>
     </property>
     <property name="buddy">
      <cstring>siteAgent</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="bytesPerKeyLbl">
     <property name="text">
      <string>Bytes Per Key</string>
     </property>
     <property name="buddy">
      <cstring>bytesPerKey</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="switchNameLbl">
     <property name="text">
      <string>Switch Name</string>
     </property>
     <property name="buddy">
      <cstring>switchName</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="bytesPerKey">
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QLabel" name="idLbl">
     <property name="text">
      <string>ID</string>
     </property>
     <property name="buddy">
      <cstring>id</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sideLbl">
     <property name="text">
      <string>Side</string>
     </property>
     <property name="buddy">
      <cstring>side</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="switchPort"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="switchPortLbl">
     <property name="text">
      <string>Switch Port</string>
     </property>
     <property name="buddy">
      <cstring>switchPort</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="kindLbl">
     <property name="text">
      <string>Device Kind</string>
     </property>
     <property name="buddy">
      <cstring>kind</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="kind"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="controlAddressLbl">
     <property name="text">
      <string>Address</string>
     </property>
     <property name="buddy">
      <cstring>controlAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="side">
     <item>
      <property name="text">
       <string>Alice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bob</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="id"/>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QLineEdit" name="siteAgent"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="controlAddress"/>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="exportConfig">
     <property name="text">
      <string>Export Config</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cqp::DeviceEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cqp::DeviceEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
